docs/wallet-integration-guide/src/party-management/index.rst

```diff
@@ -68,27 +101,64 @@ The three transactions that needs to be generated are:
 - KeyToParty: This transaction indicates that the key (public key) is associated with the party.
 
 Once all the transactions are built they can be combined into a single hash and submitted as part of a single signature.
-The wallet SDK has helper functions to generate these transactions:
+Below are examples of how to generate the topology transactions:
+
+.. tabs::
+
+    .. tab:: JS SDK
+
+      .. literalinclude:: ../../examples/snippets/create-topology-transactions.ts
+         :language: typescript
+         :dedent:
 
-.. literalinclude:: ../../examples/snippets/create-topology-transactions.ts
-   :language: typescript
-   :dedent:
+    .. tab:: Python
+
+      .. literalinclude:: ../../examples/snippets/create-topology-transactions.py
+         :language: python
+         :dedent:
+
+
+.. Do we have a bash example?
 
 Sign multi-hash
 -----------------
 Since the topology transactions need to be submitted together the combined hash needs to be signed.
 The wallet SDK has a helper function to sign the combined hash:
 
-.. literalinclude:: ../../examples/snippets/sign-party-transaction-hash.ts
-   :language: typescript
-   :dedent:
+.. tabs::
+
+    .. tab:: JS SDK
+
+      .. literalinclude:: ../../examples/snippets/sign-party-transaction-hash.ts
+         :language: typescript
+         :dedent:
+
+    .. tab:: Python
+
+      .. literalinclude:: ../../examples/snippets/sign-party-transaction-hash.py
+         :language: python
+         :dedent:
+
+.. Do we have a bash example?
 
 Submit the topology transactions
 ---------------------------------
 Once the signature is generated, the topology transactions can be submitted to the validator.
 The wallet SDK has a helper function to submit the transactions:
 
 
-.. literalinclude:: ../../examples/snippets/submit-signed-topology-transaction.ts
-   :language: typescript
-   :dedent:
+.. tabs::
+
+    .. tab:: JS SDK
+
+      .. literalinclude:: ../../examples/snippets/submit-signed-topology-transaction.ts
+         :language: typescript
+         :dedent:
+
+    .. tab:: Python
+
+      .. literalinclude:: ../../examples/snippets/submit-signed-topology-transaction.py
+         :language: python
+         :dedent:
+
+.. Again, do we have a bash example?
```

Comment:
A question for Thibault

Reply:
See https://github.com/hyperledger-labs/splice-wallet-kernel/pull/312/files#r2307723243